jscomp/bsb/bsb_ninja_check.ml

```diff
@@ -63,16 +64,18 @@ let pp_check_result fmt (check_resoult : check_result) =
         "Bsc or bsb version mismatch"
       | Bsb_forced ->
         "Bsb forced rebuild"
+      | Bsb_different_backend ->
+        "Bsb called with a different backend argument"
       | Other s -> s)
 
-let rec check_aux cwd (xs : string array) (ys: float array) i finish =
+let rec check_aux cwd ~nested (xs : string array) (ys: float array) i finish =
   if i = finish then Good
```

Comment:
what's `nested`? can we make the diff as small as possible

Reply:
Good catch, that was very useless. I used your change to make this less intrusive (I think) by ~200 lines :D